presto-orc/src/main/java/com/facebook/presto/orc/reader/ListSelectiveStreamReader.java

```diff
@@ -572,7 +572,14 @@ public Block getBlock(int[] positions, int positionCount)
             nextPosition = positions[positionIndex];
         }
 
-        return ArrayBlock.fromElementBlock(positionCount, Optional.ofNullable(nullsCopy), offsetsCopy, elementStreamReader.getBlock(elementOutputPositions, elementOutputPositionCount));
+        Block elementBlock;
```

Comment:
This change doesn't seem belong to this PR?

Reply:
there will be test failures without this change